### ISSUE TITLE
System icon support

### ISSUE THREAD

**Issue by Alchemi1963** [winapps-org/winapps#109]
I noticed that the install script favours system icons vs the present .svg files.
However, I didn't have the icons installed myself.

Eventually I edited the script to use the svg instead.
Maybe a check for the system icon can be added or let the user decide?

**Comment by Alchemi1963** [comment]
Why was legacy added?
I am **not** using legacy winapps...

**Comment by jrevillard** [comment]
legacy means the bash script.... seems that a Go version is under preparation ;)

**Comment by oskardotglobal** [comment]
*Rust, but yes

**Comment by oskardotglobal** [comment]
Legacy refers to the shell script version, which as you saw is kind of hard to maintain

**Comment by LDprg** [comment]
@NaniNoni Yeah I have some unmerged changes on my fork, but I am currently very busy.

It is still beeing worked on. See #195

**Comment by NaniNoni** [comment]
No problem. I'm happy to help in any way that I can.
